docs: Refactor condition messages and reasons (#1099)
kyma-project/telemetry-manager: docs/user/03-traces.md

          - name: "kyma-traces"
          randomSamplingPercentage: 60
    ```

### Buffer Filling Up

>   Cause: The backend sizing is inadequate

Reviewer: ```suggestion
  Cause: The backend sizing is too small for the data influx.
```

>   Cause: The backend sizing is inadequate

Reviewer 2: The word "sizing" might be a bit misleading here. Usually the problem is the possible ingestion rate, not storage capacity.

  Remedy:
  

>   - Option 1: Increase ingestion rate capabilities in your backend (i.e. scale out SAP Cloud Logging or Dynatrace instances)

Reviewer: ```suggestion
  - Option 1: Increase ingestion rate capabilities in your backend. For example, scale out the SAP Cloud Logging or Dynatrace instances.
```

>   - Option 1: Increase ingestion rate capabilities in your backend (i.e. scale out SAP Cloud Logging or Dynatrace instances)

Reviewer 2: I would remove Dynatrace here. The possibility to scale here might be limited to users who run their own on-prem cluster (which should be the exception) and more confusing for anyone else.

>   - Option 2: Decrease data influx (i.e. re-configure the trace pipeline)

Reviewer: ```suggestion
  - Option 2: Decrease data influx, that is, reconfigure the trace pipeline.
```

### Gateway Throttling 

>   Cause: Gateway is unable to receive traces at the given rate

Reviewer: ```suggestion
  Cause: Gateway cannot receive traces at the given rate.
```

  Remedy:

>   - Manually scale out the gateway (i.e. increase the number of replicas for the `telemetry-trace-gateway`). See [Module Configuration](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration)

Reviewer: ```suggestion
  - Manually scale out the gateway by increasing the number of replicas for the `telemetry-trace-gateway`. See [Module Configuration](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration).
```
